src/helpers/helpers.ts

```diff
@@ -12,3 +12,10 @@ export const isWalletLableInUse = (value: string): boolean => {
   const walletLabels = BlueApp.getWallets().map((wallet: Wallet) => wallet.label) || [];
   return walletLabels.includes(value);
 };
+
+export const getWalletTypeByLabel = (label: string): string => {
+  const wallets = BlueApp.getWallets();
+  return wallets?.find(item => {
+    return item.label === label;
+  }).type;
```

Comment:
ts doesn't shout here that find can return undefined so you cannot get the type of undefined?

Reply:
No